src/Hat.java

```diff
@@ -0,0 +1,12 @@
+/**
+ * Created by strejolopez on 11/28/16.
+ */
+
+// Hat information
+public class Hat {
+    String hatType;
+    String color;
+    int size;
+    boolean doesItLightUp;
```

Comment:
Again don't stick it into the variable name.  `doesLightUp` or something like `isLightable`